actors/miner/tests/util.rs

```diff
@@ -204,8 +261,1024 @@ impl ActorHarness {
+        let mut deadline = self.get_deadline_info(rt);
+        while deadline.last() < epoch {
+            self.advance_deadline(rt, CronConfig::empty());
+            deadline = self.get_deadline_info(rt);
+        }
+        rt.epoch = epoch;
+    }
+
+    pub fn advance_to_deadline(&self, rt: &mut MockRuntime, dlidx: u64) -> DeadlineInfo {
+        let mut dlinfo = self.deadline(rt);
+        while dlinfo.index != dlidx {
+            dlinfo = self.advance_deadline(rt, CronConfig::empty());
+        }
+        dlinfo
+    }
+
+    fn deadline(&self, rt: &MockRuntime) -> DeadlineInfo {
+        let state = self.get_state(rt);
+        state.recorded_deadline_info(&rt.policy, rt.epoch)
+    }
+
+    pub fn advance_deadline(&self, rt: &mut MockRuntime, mut cfg: CronConfig) -> DeadlineInfo {
+        let state = self.get_state(rt);
+        let deadline = new_deadline_info_from_offset_and_epoch(
+            &rt.policy,
+            state.proving_period_start,
+            rt.epoch,
+        );
+
+        if state.deadline_cron_active {
+            rt.epoch = deadline.last();
+            cfg.expected_enrollment = deadline.last() + rt.policy.wpost_challenge_window;
+            self.on_deadline_cron(rt, cfg);
+        }
+        rt.epoch = deadline.next_open();
+
+        let state = self.get_state(rt);
+        state.deadline_info(&rt.policy, rt.epoch)
+    }
+
+    fn on_deadline_cron(&self, rt: &mut MockRuntime, cfg: CronConfig) {
+        let state = self.get_state(rt);
+        rt.expect_validate_caller_addr(vec![*STORAGE_POWER_ACTOR_ADDR]);
+
+        // preamble
+        let mut power_delta = PowerPair::zero();
+        match cfg.detected_faults_power_delta {
+            Some(pwr) => {
+                power_delta += &pwr;
+            }
+            None => (),
+        };
+        match cfg.expired_sectors_power_delta {
+            Some(pwr) => {
+                power_delta += &pwr;
+            }
+            None => (),
+        };
+
+        if !power_delta.is_zero() {
+            let params = UpdateClaimedPowerParams {
+                raw_byte_delta: power_delta.raw,
+                quality_adjusted_delta: power_delta.qa,
+            };
+            rt.expect_send(
+                *STORAGE_POWER_ACTOR_ADDR,
+                PowerMethod::UpdateClaimedPower as u64,
+                RawBytes::serialize(params).unwrap(),
+                TokenAmount::from(0),
+                RawBytes::default(),
+                ExitCode::Ok,
+            );
+        }
+
+        let mut penalty_total = TokenAmount::from(0);
+        let mut pledge_delta = TokenAmount::from(0);
+
+        penalty_total += cfg.continued_faults_penalty.clone();
+        penalty_total += cfg.repaid_fee_debt.clone();
+        penalty_total += cfg.expired_precommit_penalty.clone();
+
+        if penalty_total != TokenAmount::from(0) {
+            rt.expect_send(
+                *BURNT_FUNDS_ACTOR_ADDR,
+                0, // builtin0.MethodSend
```

Comment:
nit: METHOD_SEND should be available for import and use

Reply:
done.